px-map-behavior-layer-IMS.es6.js

```diff
@@ -531,9 +731,27 @@
       const detail = {};
       if (evt.target && evt.target.feature) {
         detail.feature = evt.target.feature;
+        detail.layerId = this.id;
       }
       this.fire('px-map-layer-geojson-feature-tapped', detail);
     },
+
+    _handleIconCanvasClicked(evt) {
+      const conPoint = evt.containerPoint;
+      const points = this._iconTree.search({
+        minX: conPoint.x,
+        minY: conPoint.y,
+        maxX: conPoint.x,
+        maxY: conPoint.y
+      });
+      // TODO - return array of features
```

Comment:
Is this TODO done?

Reply:
I'll remove the todo - the event signature is only for one feature at the moment.